### PR TITLE
Update minsdk android

### PR DESCRIPTION
**Pull request by tmoldovan8x8**

### CODE REVIEW

File: spot-controller/ios/spot-controller/AppDelegate.m [hunk 1]
```diff
@@ -6,28 +6,17 @@
...
-#import <React/RCTPushNotificationManager.h>
 #import <React/RCTRootView.h>
 
-@import Firebase;
-
 @implementation AppDelegate
 
 - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(NSDictionary *)launchOptions
 {
-  // Initialize Crashlytics and Firebase if a valid GoogleService-Info.plist file was provided.
```
Comment (saghul) [inline spot-controller/ios/spot-controller/AppDelegate.m:21]: Why are we removing this?

**Review by saghul — COMMENTED** [review]
2 comments, LGTM otherwise.

**Comment by saghul** [comment]
@dudumanbogdan Does the test failure sound familiar?

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
Pl try to look with @dudumanbogdan why those tests fail now.

**Comment by dudumanbogdan** [comment]
> @dudumanbogdan Does the test failure sound familiar?

No :(. I saw the reason was a time out, not sure if it's related to github machine or is something from the changes `Error: Timed out waiting for: http://localhost:8000/`

We should see with next PR if this still continues to failed or not

**Pull request merged**